observations/obs_converters/README.md

```diff
@@ -279,15 +279,21 @@ The *DART/observations/obs\_converters* directory contains a variety of
 converter programs to read various external formats and convert the
```

Comment:
git is too smart for me.  i wanted to make a note in a part of this file that hasn't changed and i can't figure out how to expand the file do it.   (it keeps showing me the formatted markdown!)  </whine>   

in the initial 'overview' section, the second-to-last paragraph, it says ALL converters use 3d sphere locations.  that's not quite true (there are a couple that generate 1d, and possibly 3d cartesian).  could we make that say "Most ..." instead?

Reply:
done.